### PR TITLE
Change JS timezone to milliseconds and expose timezone hours and minutes in API

### PR DESCRIPTION
**Pull request by megri**

### CODE REVIEW

**Comment by darkfrog26** [comment]
@megri a good start, but did you see that Scala Native is failing in Travis?

File: core/shared/src/main/scala/perfolation/CrossDate.scala [hunk 1]
```diff
@@ -34,7 +36,10 @@ trait CrossDate {
   def L: String = int(milliOfSecond, 3)
   def p: String = if (isAM) "am" else "pm"
   def P: String = if (isAM) "AM" else "PM"
-  def z: String = int(timeZoneOffset, 4)
+  def z: String = {
+    val sign = if (timeZoneOffsetMillis >= 0) "+" else "-"
+    sign + timeZoneOffsetHH + timeZoneOffsetMM
```
Comment (darkfrog26) [inline core/shared/src/main/scala/perfolation/CrossDate.scala]: You should really change this to `p"$sign$timeZoneOffsetHH$timeZoneOffsetMM"`

Comment (megri) [reply core/shared/src/main/scala/perfolation/CrossDate.scala]: Good call.

**Comment by darkfrog26** [comment]
Great work!

**Comment by darkfrog26** [comment]
This has released in 1.0.1

**Pull request merged**